docs/pages/example/mapbox-gl-geocoder-custom-render.html

```diff
@@ -0,0 +1,39 @@
+<script src='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v4.0.0/mapbox-gl-geocoder.min.js'></script>
+<link rel='stylesheet' href='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v4.0.0/mapbox-gl-geocoder.css' type='text/css' />
+<style>
+.geocoder-dropdown-item{
+    padding: 5px
+}
+.geocoder-dropdown-icon{
+    margin-right: 10px;
+}
+.geocoder-dropdown-text{
+    color: darkgray;
```

Comment:
This color value's contrast is very low, which will make it difficult for users with low vision to read. Maybe go with a darker gray? Or a fun purple `#5a3fc0` ?

```suggestion
    color: #666;
```